### PR TITLE
PR conventions draft.

### PR DESCRIPTION
**Pull request by syntheticmagus**

### CODE REVIEW

File: Documentation/PullRequestConventions.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Conventions for Pull Requests
+
+At present, the Babylon Native repository predominantly relies on an 
+"honor system" when it comes to pull requests, placing the bulk of the 
+responsibility for making sure the PR process remains effective and 
+efficient overwhelmingly in the hands of those submitting and performing 
+the code reviews. Consequently, this repo does not currently have strong 
+established practices for how pull requests, but we do have a number of
```
Comment (rickfromwork) [inline Documentation/PullRequestConventions.md:8]: for how pull requests?

File: Documentation/PullRequestConventions.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+At present, the Babylon Native repository predominantly relies on an 
+"honor system" when it comes to pull requests, placing the bulk of the 
+responsibility for making sure the PR process remains effective and 
+efficient overwhelmingly in the hands of those submitting and performing 
+the code reviews. Consequently, this repo does not currently have strong 
+established practices for how pull requests, but we do have a number of 
+conventions which we typically try to follow when performing and receiving 
+code reviews. This document describes these conventions; it is formatted as 
+a series of  definitions for various actions and intended usages in a
```
Comment (rickfromwork) [inline Documentation/PullRequestConventions.md:11]: A few spots with double spaces, "of  definitions" and "Babylon  Native" in the next line

**Review by rickfromwork — APPROVED** [review]

**Review by bghgary — APPROVED** [review]

**Comment by CedricGuillemet** [comment]
Is it the right document to talk about running validation tests on platforms not supported by the CI like iOS?

**Comment by bghgary** [comment]
Can we merge this?

**Pull request merged**